src/oracle-database/azext_oracle_database/aaz/latest/oracle_database/cloud_exadata_infrastructure/db_server/__cmd_group.py

```diff
@@ -0,0 +1,23 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command_group(
+    "oracle-database cloud-exadata-infrastructure db-server",
```

Comment:
I found you are using two similar meaning keywords `database` and `db` while naming the commands. It's better to choice only one.